### DISCUSSION TITLE
How can I make the banner full width with `title-block-banner: true`?

### DISCUSSION THREAD

**Discussion by cbrnr** [quarto-dev/quarto-cli#7574] [Q&A] [answered]
### Description

I'm using the full HTML layout to make the rendered text a bit wider. I also like the title block banner. Therefore, I include the following options in my YAML header:

````qmd
---
title: "Hello Quarto!"
format:
  html:
    page-layout: full
author: Some Random Person
published: 2023-11-14
title-block-banner: true
---

## First section
This is just some sample text to demonstrate how it gets rendered using the full HTML page layout.
It's a little wider than the default layout, so more text fits on a page!
````

Now my problem is that the banner fields are not aligned with the main text, i.e. the left border seems to be at the regular (as opposed to full) layout position. How can I make the banner fields full width?

![Screenshot 2023-11-14 at 15 20 32](https://github.com/quarto-dev/quarto-cli/assets/4377312/9b65a6d3-c957-4a6f-a91c-2619491c8b6b)

**Comment by mcanouil** [comment, marked as answer]
Either modify the title bloc partial (see <https://quarto.org/docs/journals/templates.html>) or modify the CSS rule (use your favorite browser inspect/developper mode to know what to target).

Note that using the latest version (pre-release) you don't get the output of your screenshot.

**Reply by cbrnr** [reply]
Thank you! So this has already been fixed in the pre-release – great!
